Z80_CPU.cpp

```diff
@@ -141,8 +141,19 @@ void Z80_CPU::LD_IMMEDIATE_8BIT_DATA_TO_8BIT_REGISTER(int8_t &reg)
 	WR_REGISTER(REGISTER_ACCESS_MODE::WRITE, reg);
 }
 
-/* 
-	Load 8-bit data in 8-bit register to absolute address in (HL)
+/*
+	Load to the 8-bit register r, data from the absolute address specified by the 16-bit register HL.
+*/
+
+void Z80_CPU::LD_ABS_ADDR_IN_HL_TO_8BIT_REGISTER(int8_t &reg)
```

Comment:
We're not loading Absolute address in HL to 8-bit register, we're loading **data** in absolute address in register HL.